roles/setup_dbt2/tasks/dbt2_install_packages_driver.yml

```diff
@@ -42,14 +13,25 @@
     mode: '0755'
   become: true
 
-- name: Install additional supporting packages
+- name: Install DBT-2 Driver supporting packages
   ansible.builtin.package:
     name:
```

Comment:
don't need python3-docutils anymore, it's in the appimage.  also don't need postgresql anymore, that's also in the appimage.  also don't need dbttools, that's in the appimage.